optimum/intel/neural_compressor/neural_coder_adaptor.py

```diff
@@ -0,0 +1,26 @@
+class NeuralCoderAdaptor:
+    """API design adaption for Neural Coder"""
+
+    default_quant_dynamic = f"""\
+def eval_func(model):
+    EVAL_FUNC_LINES
+from neural_compressor.config import PostTrainingQuantConfig
+from optimum.intel.neural_compressor import INCQuantizer
+quantization_config = PostTrainingQuantConfig(approach="dynamic")
+quantizer = INCQuantizer.from_pretrained(MODEL_NAME)
+quantizer.quantize(quantization_config=quantization_config, save_directory="quantized_model", save_onnx_model=False)
+MODEL_NAME = quantizer._quantized_model
+MODEL_NAME.eval()
+"""
+
+    default_quant_static = f"""\
+def eval_func(model):
+    EVAL_FUNC_LINES
+from neural_compressor.config import PostTrainingQuantConfig
+from optimum.intel.neural_compressor import INCQuantizer
+quantization_config = PostTrainingQuantConfig(approach="static")
+quantizer = INCQuantizer.from_pretrained(MODEL_NAME)
+quantizer.quantize(quantization_config=quantization_config, calibration_dataset=eval_dataset, save_directory="quantized_model", save_onnx_model=False)
+MODEL_NAME = quantizer._quantized_model
+MODEL_NAME.eval()
```

Comment:
Why is it necessary ?

Reply:
Are you referring to line 25, or?

Reply:
Yes !

Reply:
This line converts model to eval (evaluation) mode for PyTorch inference, we added this so that it's clear that the model will be in eval mode for the rest part of the code. Because quantized mode can only be used in evaluation mode not training mode (as far as I know)

Reply:
Ok this was added in case the model is used after the quantization step, thanks for the explanation !